<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalPage</class>
 <widget class="QWidget" name="TerminalPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>488</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Terminal&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line14">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TextLabel13_2_2_2_2">
       <property name="text">
        <string>Linenumbers width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="linenowidthSlider">
       <property name="toolTip">
        <string>Move to set the width of the linenumbers margin.</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="linenowidthLCD">
       <property name="toolTip">
        <string>Displays the selected width of the linenumbers margin.</string>
       </property>
       <property name="numDigits">
        <number>1</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="linenoCheckBox">
       <property name="toolTip">
        <string>Select whether line numbers margin should be shown.</string>
       </property>
       <property name="text">
        <string>Show Line Numbers Margin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="syntaxHighlightingCheckBox">
       <property name="toolTip">
        <string>Select to enable syntax highlighting</string>
       </property>
       <property name="text">
        <string>Syntax Highlighting Enabled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1_20">
       <property name="text">
        <string>max. History Entries:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="historySpinBox">
       <property name="toolTip">
        <string>Enter the number of history entries allowed</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="monospacedFontButton">
        <property name="toolTip">
         <string>Press to select the font to be used as the monospaced font</string>
        </property>
        <property name="text">
         <string>Monospaced Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="monospacedFontSample">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Monospaced Text</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="monospacedCheckBox">
        <property name="toolTip">
         <string>Select, whether the monospaced font should be used as default</string>
        </property>
        <property name="text">
         <string>Use monospaced as default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="linenumbersFontButton">
        <property name="toolTip">
         <string>Press to select the font for the line numbers</string>
        </property>
        <property name="text">
         <string>Line Numbers Font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="marginsFontSample">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>2345</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shellGroup">
     <property name="title">
      <string>Shell</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shell command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="shellCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter the shell to be used</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="interactiveCheckBox">
        <property name="toolTip">
         <string>Select to start an interactive shell</string>
        </property>
        <property name="text">
         <string>Interactive Shell</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>linenowidthSlider</tabstop>
  <tabstop>linenoCheckBox</tabstop>
  <tabstop>syntaxHighlightingCheckBox</tabstop>
  <tabstop>historySpinBox</tabstop>
  <tabstop>monospacedFontButton</tabstop>
  <tabstop>monospacedCheckBox</tabstop>
  <tabstop>linenumbersFontButton</tabstop>
  <tabstop>shellCombo</tabstop>
  <tabstop>interactiveCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>linenowidthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>linenowidthLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
